### ISSUE TITLE
Searching using the global search crashed computer due to searching in previously excluded directories

### ISSUE THREAD

**Issue by baalimago** [microsoft/vscode#99829]
- VSCode Version: 1.46.0
- OS Version: Ubuntu 18.04 LTS

Steps to Reproduce:

1. Update to 1.46.0
2. Expect vs code searching to work like it did prior to the update and be annoyingly surprised at how global searches (Ctrl+shift+f window) now will crash your computer  as it relentlessly attempts to find strings in previously excluded folders.

After the update, for some reason, the searches the project differently from before. Now it's scanning through dist and coverage folders in the project, which it didn't use to, and there doesn't seem to be a way to make it stop either once it's started. I had to manually restart the computer, after spending a good 5-10 min having a lagged out computer. I looked through the patch notes, nothing was mentioned about changing the search exclusion policy (or whatever happened).

Maybe it has to do with 'exclude settings and ignore files are disabled'? I don't know. And I don't know how to enable exclude settings and ignore files, it used to be on as default. This issue didn't exist before the update.

The "Search Editor" still works as before, though, thankfully. 

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
Sorry for the trouble. I am investigating the issue that causes hangs, it's been reported but I haven't been able to repro it yet, see https://github.com/microsoft/vscode/issues/88385.

> exclude settings and ignore files are disabled

It sounds like you disabled that option, make sure this button is toggled on

![image](https://user-images.githubusercontent.com/323878/84609345-4846d200-ae7c-11ea-92e7-0a88f20c3b32.png)

**Comment by gclements-chwy** [comment]
I ran into this too. Things are fine for a little while. Then I do a global search and keys stop entering into the text box and it's time for a hard reboot. The latest incident: I read this ticket, started VS Code, went to global search, I had `if (usePares) {` from a previous search, I clicked on the gear next to files to exclude and everything lagged and it's time for a hard reboot. This is a deal breaker for me. For now I'm going to try the insider's version.
